securedrop/tests/test_journalist.py

```diff
@@ -760,8 +906,8 @@ def test_admin_renames_user(self):
         # therefore asserting it has been created
         Journalist.query.filter(Journalist.username == new_user).one()
 
-    @patch('journalist_app.admin.current_app.logger.error')
-    @patch('journalist_app.admin.Journalist',
+    @patch('journalist_app_module.admin.current_app.logger.error')
+    @patch('journalist_app_module.admin.Journalist',
```

Comment:
As is, this patching should be applied to `journalist_app` e.g. `@patch('journalist_app.admin.Journalist', ...`